pkgs/development/php-packages/luasandbox/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildPecl,
+  fetchFromGitHub,
+  pkg-config,
+  lua51Packages,
+}:
+
+buildPecl rec {
+  pname = "luasandbox";
+  version = "4.1.2";
+  src = fetchFromGitHub {
+    owner = "wikimedia";
+    repo = "mediawiki-php-luasandbox";
+    tag = version;
+    hash = "sha256-HWObytoHBvxF9+QC62yJfi6MuHOOXFbSNkhuz5zWPCY=";
+  };
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ lua51Packages.lua ];
+  meta = with lib; {
```

Comment:
Please get rid of `with lib;`, it makes evaluation run slower. Do it here and in the other PRs you submitted.

Thank you.